docs/en/ufs/S3.md

```diff
@@ -6,8 +6,6 @@ group: Under Stores
 priority: 0
 ---
 
-TODO(calvin, lu): Update content
```

Comment:
is this addressed? @calvinjia @LuQQiu ?

Reply:
I didn't notice this comment, I will take a look this morning.

Reply:
@apc999 all the s3 contents are up-to-date(S3 streaming upload included). I am not sure what else can be added.
@calvinjia any ideas on what is missing?

Reply:
We decided after the TODO was created that we would not to make major content changes to the UFS docs

Reply:
ack